src/components/Input/Input.tsx

```diff
@@ -56,6 +59,7 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>((props, ref?) => {
     let inputProps = {
         id: id ? `input-${id}` : null,
         className: bem("input", modifiers, blockName, [className]),
+        defaultValue: defaultValue,
         type: type,
         value: value,
```

Comment:
There's already a `value` attribute in this object that will go into the `input` element. I haven't tested this yet but it'll probably complain about the controlled vs uncontrolled component type. I think if you're adding a `defaultValue` then you also need an `onChange` function similar to PR #413 but I'll test it.